aqt/metadata.py

```diff
@@ -24,6 +24,7 @@
 import re
 import secrets as random
 import shutil
+from urllib.parse import urlparse
```

Comment:
This part of the 'check' action: https://github.com/miurahr/aqtinstall/runs/6331277720?check_suite_focus=true#step:7:17 failed because the imports are no longer sorted. You can fix that with isort.

If this helps, I always run `isort aqt tests && black aqt tests` before I commit anything that's going into this codebase, just to avoid any code formatting or import sorting issues. I think you can fix them with `flake8` as well.